lib/qa/authorities/linked_data/config/term_config.rb

```diff
@@ -222,16 +240,28 @@ def info
 
       private
 
-        def pred_id_from_ldpath
+        def id_predicates_from_ldpath
           # prefix example: { skos: 'http://www.w3.org/2004/02/skos/core#' }
           # ldpath example: 'skos:id :: xsd:string'
+          predicates = []
```

Comment:
Might be worth trying dropping 246/248/254 and change 250 to be a map. If that doesn't work because of empty strings or something, it's probably not worth it.